### DISCUSSION TITLE
Upvote hover shadow position.

### DISCUSSION THREAD

**Discussion by MrMohamedAbdallah** [dailydotdev/daily#1042] [Feature Requests]
I want to suggest the shadow for the upvote button to be changed to be on the top to match the arrow direction. And the down vote button to stay as it's.

Example:
![image](https://github.com/dailydotdev/daily/assets/51543900/dc6ff7f4-94b3-423f-ad03-20052e72cce1)

**Comment by idoshamun** [comment]
Hi @MrMohamedAbdallah,

Thanks for the suggestion! Unfortunately, this is not aligned with our design guidelines and cannot be approved 🙏
